p2p/libp2p/netMessenger.go

```diff
@@ -1198,6 +1200,10 @@ func (netMes *networkMessenger) GetConnectedPeersInfo() *p2p.ConnectedPeersInfo
 			connPeerInfo.IntraShardObservers[peerInfo.ShardID] = append(connPeerInfo.IntraShardObservers[peerInfo.ShardID], connString)
 			connPeerInfo.NumIntraShardObservers++
 		}
+
+		if netMes.peersHolder.Contains(pid) {
```

Comment:
could have renamed netMes.peersHolder to netMes.preferredPeersHolder (it has become obvious only now)

Reply:
done